e2e/wdio/core/tests/tile.e2e-spec.ts

```diff
@@ -0,0 +1,114 @@
+import { TilePo } from '../pages/tile.po';
+import {
+    getText,
+    refreshPage,
+    getElementClass,
+    scrollIntoView,
+    getElementArrayLength,
+    setValue,
+    getValue,
+    getAttributeByName,
+    getElementPlaceholder,
+    mouseHoverElement,
+    isElementDisplayed,
+    isElementClickable,
+    click,
+    sendKeys,
+    waitForPresent,
+    acceptAlert,
+} from '../../driver/wdio';
+
+import { sections, states } from '../fixtures/appData/textarea-contents'
```

Comment:
don't forget to delete unused imports
![2021-08-10_1](https://user-images.githubusercontent.com/47522152/128844504-80bb920b-6909-441e-8435-f6f701b33de4.png)